pom.xml

```diff
@@ -268,10 +268,39 @@
           <release>11</release>
         </configuration>
       </plugin>
+      <plugin>
```

Comment:
@kurtnordstrom is there a particular reason why this change is in this PR (not a problem, just curious)?

Reply:
Primarily for checking test coverage.